src/Tickets/Commerce/Hooks.php

```diff
@@ -123,6 +124,8 @@ protected function add_filters() {
 		add_filter( 'tribe_template_context:tickets-plus/v2/tickets/submit/button-modal', [ $this, 'filter_showing_cart_button' ] );
 
 		add_filter( 'tec_tickets_commerce_payments_tab_settings', [ $this, 'filter_payments_tab_settings' ] );
+		
+		add_filter( 'wp_redirect', [ tribe( Payments_Tab::class ), 'filter_redirect_url' ] );
```

Comment:
Look at how we did the method `filter_payments_tab_settings` for the hook `tec_tickets_commerce_payments_tab_settings`, you need to create an alias method in this hooks class.